src/client/datascience/notebookStorage/vscNotebookModel.ts

```diff
@@ -99,13 +99,15 @@ export class VSCodeNotebookModel extends BaseNotebookModel {
         if (!this.document) {
             return [];
         }
-        return this.document.cells.map((cell) => {
-            return {
-                id: cell.document.uri.toString(),
-                data: createJupyterCellFromVSCNotebookCell(cell),
-                state: cellRunStateToCellState(cell.metadata.runState)
-            };
-        });
+        return this.document.cells
+            .filter((cell) => !cell.document.isClosed)
```

Comment:
In one of @DavidKutu PR i noticed this failing.
Ignore cells that have been closed/deleted.